test/unit/app/api-requests/users.test.js

```diff
@@ -61,4 +61,15 @@ describe('Get users', () => {
 
     expect(res).toEqual(JSON.parse(fileContent)[0])
   })
+
+  test('return user data when test email is matched but has different casing', async () => {
```

Comment:
Consider making this test a test case and combine it with the one above - https://jestjs.io/docs/api#testeachtablename-fn-timeout